llvm/lib/Target/AMDGPU/AMDGPUAttributor.cpp

```diff
@@ -1365,9 +1366,30 @@ static bool runImpl(Module &M, AnalysisGetter &AG, TargetMachine &TM,
 
   Attributor A(Functions, InfoCache, AC);
 
-  LLVM_DEBUG(dbgs() << "[AMDGPUAttributor] Module " << M.getName() << " is "
-                    << (AC.IsClosedWorldModule ? "" : "not ")
-                    << "assumed to be a closed world.\n");
+  LLVM_DEBUG({
+    StringRef LTOPhaseStr;
+    switch (LTOPhase) {
+    case ThinOrFullLTOPhase::None:
+      LTOPhaseStr = "None";
+      break;
+    case ThinOrFullLTOPhase::ThinLTOPreLink:
+      LTOPhaseStr = "ThinLTOPreLink";
+      break;
+    case ThinOrFullLTOPhase::ThinLTOPostLink:
+      LTOPhaseStr = "ThinLTOPostLink";
+      break;
+    case ThinOrFullLTOPhase::FullLTOPreLink:
+      LTOPhaseStr = "FullLTOPreLink";
+      break;
+    case ThinOrFullLTOPhase::FullLTOPostLink:
+      LTOPhaseStr = "FullLTOPostLink";
+      break;
+    }
```

Comment:
Move this to a switch enum -> return name utility function, preferably along side where the enum is defined